test/ampycloud/ref_data/Geneva_2020.10.22.-21.15.00_SCT004-SCT007.csv

```diff
@@ -0,0 +1,249 @@
+,ceilo,dt,alt,type
+0,Ceilometer.Final05,-886.0,570.0,1
```

Comment:
Same as previously. Would be good to remove the indexes.

Reply:
Done, thanks!